graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/functions/messages/CloneMessage.java

```diff
@@ -56,7 +90,7 @@ public Message evaluate(FunctionArgs args, EvaluationContext context) {
     public FunctionDescriptor<Message> descriptor() {
         return FunctionDescriptor.<Message>builder()
                 .name(NAME)
-                .params(ImmutableList.of(messageParam))
+                .params(ImmutableList.of(loopDetectionParam, messageParam))
```

Comment:
```suggestion
                .params(ImmutableList.of(messageParam, loopDetectionParam))
```

I think we need to add the new parameter at the end. Otherwise, existing usages of the rule will break with an error like this:

```
2023-02-22 11:43:13,724 WARN : org.graylog.plugins.pipelineprocessor.processors.ConfigurationStateUpdater - Ignoring non parseable rule <clone/63f5ec78b3a3156cd887fe6c> with errors <[Expected type Boolean for argument preventLoops but found Message in call to function clone_message in line 5 pos 4]>